service-data-access/src/test/java/gov/va/vro/abddataaccess/service/FhirClientTest.java

```diff
@@ -89,24 +89,24 @@ private void mockGetBundle(Bundle bundle, AbdDomain domain) throws AbdException
           .getBundle(domain, TEST_PATIENT, DEFAULT_PAGE + 1, DEFAULT_SIZE);
     }
   }
-
-  @Test
-  public void testGetMedicalEvidence() {
-    AbdClaim testClaim = new AbdClaim();
-    testClaim.setClaimSubmissionId(TEST_CLAIM_ID);
-    testClaim.setDiagnosticCode(TEST_MEDICATION_REQUEST);
-    testClaim.setVeteranIcn(TEST_PATIENT);
-    try {
-      mockGetBundle(medicationBundle, AbdDomain.MEDICATION);
-      AbdEvidence evidence = client.getMedicalEvidence(testClaim);
-      assertNotNull(evidence);
-      assertTrue(evidence.getMedications().size() > 0);
-      assertEquals(evidence.getMedications().size(), medicationBundle.getEntry().size());
-    } catch (Exception e) {
-      log.error("testGetMedicalEvidence error: {}", e.getMessage(), e);
-      fail("testGetMedicalEvidence");
-    }
-  }
+  //
```

Comment:
@wlin2019 How could we fix this test, now that I am adding back in Condition to the 6602 claim data

Reply:
I wonder whether this branch has been merged with the develop branch correctly or not. The test directories look different. I'll try to fix it.

Reply:
Thanks, I will update the branch with develop.

Reply:
I updated V2-asthma branch to make service-data-access/test directory the same as the one in develop branch. All service-data-access tests are passed.